meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb

```diff
@@ -44,18 +49,48 @@ do_compile() {
 	if [ "x${UBOOT_SIGN_ENABLE}" = "x1" ]; then
 		uboot-mkimage -F -k "${UBOOT_SIGN_KEYDIR}" -r boot.itb
 	fi
+
+	if [ -n "${LMP_BOOT_FIRMWARE_FILES}" ]; then
```

Comment:
I'm not sure which one is better here: check if `LMP_BOOT_FIRMWARE_FILES` is defined or check if `update.itb` is added to `LMP_BOOT_FIRMWARE_FILES`